client/client-multi/client-c/src/kaa_context.h

```diff
@@ -28,23 +28,52 @@
 extern "C" {
 #endif
 
-typedef struct kaa_status_t             kaa_status_t;
-typedef struct kaa_platform_protocol_t  kaa_platform_protocol_t;
+#ifndef KAA_STATUS_T
+    #define KAA_STATUS_T
```

Comment:
Let us please have hash at the beginning of the line even for indented preprocessor commands:
#ifndef KAA_STATUS_T
#   define KAA_STATUS_T